### ISSUE TITLE
Cannot connect file share，cannot recognize non-ascii path

### ISSUE THREAD

**Issue by wangzyon** [opencv/cvat#4428]
develop branch
version 3c8922ec7

![image](https://user-images.githubusercontent.com/36832126/156959651-afd58964-3fe8-4e13-93b9-80ab7367843e.png)

**Comment by bsekachev** [comment]
Thank you for report, need to check

**Comment by rhee-elten** [comment]
I encountered the same problem, and found a possible fix.
To test the idea, the fix is applied onto a new fork of this project ( 3bd7300cc804874c774d430d504ebb98563d3843 )
Please check this fix works and has no other problems. Thank you.

```
diff --git a/cvat-core/src/server-proxy.ts b/cvat-core/src/server-proxy.ts
index bcc7e3ad..0dd22d4d 100644
--- a/cvat-core/src/server-proxy.ts
+++ b/cvat-core/src/server-proxy.ts
@@ -217,13 +217,12 @@
 
             async function share(directoryArg) {
                 const { backendAPI } = config;
-                const directory = encodeURI(directoryArg);
 
                 let response = null;
                 try {
                     response = await Axios.get(`${backendAPI}/server/share`, {
                         proxy: config.proxy,
-                        params: { directory },
+                        params: { directoryArg },
                     });
                 } catch (errorData) {
                     throw generateError(errorData);
```

**Comment by nmanovic** [comment]
@rhee-elten , could you please send us a PR?

**Comment by rhee-elten** [comment]
@nmanovic, I just sent a PR #4983 for above fix.

**Comment by rhee-elten** [comment]
@nmanovic , i just closed PR #4983 and re-sent an equivalent PR #4946 over 1db247aeddc0c0413f52086a1f6bde0485fc32b8 because previous #4983 conflicts with update HEAD and could not be resolved automatically.
